### ISSUE TITLE
1 hr 40 minutes for a withdraw?

### ISSUE THREAD

**Issue by Pnin1204** [Loopring/dexwebapp#108]
I tried to purchase and withdraw in order to take advantage of an arbitrage situation, and the withdraw has taken 1 hour 40 minutes, and is still pending.  I think this won't be plausible for many traders.

<img width="1329" alt="Screen Shot 2020-03-12 at 08 28 32" src="https://user-images.githubusercontent.com/35647557/76532183-de153000-643b-11ea-8775-f88b06c87391.png">

**Comment by kenpole** [comment]
Also, the fact that it still displays as a tradable asset is confusing and could lead to issues.  I still see it in my wallet balance, and I seem to be able to make orders with the token.

**Comment by dong77** [comment]
FYI, today the Ethereum network (and Bitcoin's as well) is very busy. Your experience has nothing to do with Loopring Exchange. Your request for withdrawal has not been confirmed by the Ethereum network, so it doesn't yet exist from the Exchange's perspective.

**Comment by kenpole** [comment]
I'm actually still unable to withdraw my tokens, at this point.  I get an error message.  Something about an rpc error, and the error message is "your withdraw failed, please try again" Which continues to fail.

**Comment by dong77** [comment]
Most likely your metamask did not pop up a window for you to sign tx. Please make your window not full screen and try it again. Also make sure there is pending requests in metamask. 

Even you request is successful, you’ll have to wait longer than usual, the Ethereum network is experience higher volume now.  You may monitor etherscan.io/address/loopringio.eth to see how our transactions get processed.

**Comment by dong77** [comment]
Can you share a screenshot?

**Comment by kenpole** [comment]
<img width="1440" alt="Screen Shot 2020-03-13 at 15 23 26" src="https://user-images.githubusercontent.com/35647557/76660608-e609c880-653e-11ea-9dae-be7a00af1b9a.png">
<img width="993" alt="Screen Shot 2020-03-13 at 15 22 01" src="https://user-images.githubusercontent.com/35647557/76660610-e904b900-653e-11ea-80a5-47d004b3d4a4.png">
<img width="556" alt="Screen Shot 2020-03-13 at 15 21 17" src="https://user-images.githubusercontent.com/35647557/76660611-e99d4f80-653e-11ea-989d-d17c461d6787.png">

**Comment by dong77** [comment]
I noticed the above error, but can you provide more details about the error? In the meanwhile, I’ll also try to use Brave to deposit and withdraw tokens.
I also want to confirm if you are using Brave’s built in wallet or use MetaMask on Brave. Thank you.

- Daniel

**Comment by kenpole** [comment]
I've never installed metamask on this browser, so it's the Brave's built-in wallet that looks pretty much identical to metamask.  This was during high congestion on ETH, when I noticed the price disparity between Link on Loopring, and when I tried to withdraw, everything seemed to go smoothly.  I feel like the gas limit was too low, and thus, at least within Loopring.io's system, after it seemed to be taking forever I couldn't resubmit or cancel the process, and it simply got stuck.  It wouldn't let me put in a new withdraw, and so in theory I'd have to go into myetherwallet or something and replace the transaction there, which an average user may or may not (most likely not) be able to do.  I would assume an auto-cancel would be better.

Unfortunately, this meant that in my attempt to resubmit an order, I remember something about a gas fee that I may have gotten hit by, so I didn't want to try again.  That's when I went in and recorded a video just to make sure I had it recorded.

Not sure those are the details you needed.
